http.c

```diff
@@ -2283,6 +2283,10 @@ evhttp_parse_headers_(struct evhttp_request *req, struct evbuffer* buffer)
 
 		req->headers_size += len;
 
+		if (memchr(line, '\x00', len) != NULL) {
```

Comment:
This can make the parser slower, but I guess line should not be that big, so it should be fine